execution_engine_testing/tests/src/test/get_balance.rs

```diff
@@ -132,7 +132,7 @@ fn get_balance_using_public_key_should_work() {
 
     let alice_main_purse = alice_account.main_purse();
 
-    let alice_balance_result = builder.get_public_key_balance_result(*ALICE_KEY);
```

Comment:
Was this broken on `dev`?

Reply:
Apparently, I had to merge with `dev` and resolve Clippy issues on my branch.